cylc/rose/utilities.py

```diff
@@ -283,13 +283,13 @@ def get_cli_opts_node(opts=None):
     # Unpack info we want from opts:
     opt_conf_keys = []
     defines = []
-    suite_defines = []
+    rose_template_var = []
     if opts and 'opt_conf_keys' in dir(opts):
         opt_conf_keys = opts.opt_conf_keys
     if opts and 'defines' in dir(opts):
         defines = opts.defines
-    if opts and 'define_suites' in dir(opts):
-        suite_defines = opts.define_suites
+    if opts and 'rose_template_var' in dir(opts):
+        rose_template_var = opts.rose_template_var
```

Comment:
Sorry to be horribly pedantic - can we make this plural?

Reply:
Makes sense

Reply:
https://github.com/cylc/cylc-flow/pull/4200